File: 백엔드/AMN/src/main/java/com/ibkpoc/amn/dto/MeetingListResponse.java
package com.ibkpoc.amn.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

// MeetingListResponse.java
@Data
@NoArgsConstructor
@AllArgsConstructor
public class MeetingListResponse {
    private Long id;
    private String title;
    private LocalDateTime start;
    private LocalDateTime end;
    private Integer participant;
    private String topic;
}

File: 앱/IBK_PoC/app/src/main/java/com/ibkpoc/amn/model/MeetingListResponse.kt
// model/MeetingListResponse.kt (수정)
package com.ibkpoc.amn.model

data class MeetingListResponse(
    val id: Long,
    val title: String,
    val start: String,
    val end: String,
    val participant: Int,
    val topic: String?
)